src/Fmt_ast.ml

```diff
@@ -1754,7 +1754,11 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
                                       $ fmt "@ " $ fmt_expression c xcnd )
                                   $ fmt "@ then" )
                             | None -> fmt "else" )
-                          $ fmt_if parens_bch " (" $ fmt "@ "
+                          $ fmt_or parens_bch
+                              ( if c.conf.indicate_multiline_delimiters then
+                                " (@ "
+                              else " (@," )
```

Comment:
The indentation of `else` looks weird

Reply:
Agreed but according to Josh https://github.com/ocaml-ppx/ocamlformat/pull/603#discussion_r262851548 it's the expected behavior until we add a new option for that.

Reply:
Yes, this is the expected form for `if-then-else=compact`